examples/jsdoc-project/README.md

```diff
@@ -0,0 +1,26 @@
+## Overview
+
+Example of how to use [jsdoc](http://usejsdoc.org/) in MarkLogic project
+
+This project uses the [JSDoc Gradle Plugin](https://github.com/liferay/liferay-portal/tree/master/modules/sdk/gradle-plugins-jsdoc) to run the jsdoc task
+
+
+## Usage
+
+All of the configuration is captured in the __build.gradle__ file and the __jsdoc.json__ file. To test this, clone this repoitory and execute the command below in this directory 
+
+```
+gradle mlJsDoc
```

Comment:
Can you change this to "jsDoc" or something without the "ml" prefix? The "ml" prefix implies that the task is provided by the ml-gradle plugin. 

Otherwise looks good - I'll merge after that change. Neat plugin, I like the results.

Reply:
sure no problem - i've just renamed it to generateJsDoc